### PR TITLE
CDAP-11655 Deploy and Start Wrangler in SDK#Default. Fix a bug in similar feature for Tracker

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-11655

Build :  https://builds.cask.co/browse/CDAP-RUT1040

### CODE REVIEW

**Comment by yaojiefeng** [comment]
LGTM

**Comment by gokulavasan** [comment]
@yaojiefeng I have added the logic to deploy Wrangler in this PR. Can you review the second commit?

File: cdap-standalone/src/main/java/co/cask/cdap/AbstractAppCreationService.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    applicationLifecycleService.deployApp(
+      appId.getParent(), appId.getApplication(), appId.getVersion(),
+      NamespaceId.SYSTEM.artifact(artifactSummary.getName(), artifactSummary.getVersion()).toId(),
+      appConfig, new DefaultProgramTerminator());
+
+    for (Map.Entry<ProgramId, Map<String, String>> programEntry : programIdMap.entrySet()) {
+      try {
+        programLifecycleService.start(programEntry.getKey(), programEntry.getValue(), false);
+      } catch (IOException ex) {
+        // Might happen if the program is being started in parallel through tUI
```
Comment (yaojiefeng) [inline cdap-standalone/src/main/java/co/cask/cdap/AbstractAppCreationService.java]: extra 't' in comment

**Comment by yaojiefeng** [comment]
Just one minor comment, rest LGTM

**Pull request merged**